.drone.star

```diff
@@ -482,7 +482,7 @@ def localApiTests(ctx, storage = "owncloud", suite = "apiBugDemonstration", acco
                     "TEST_OCIS": "true",
                     "STORAGE_DRIVER": storage,
                     "BEHAT_SUITE": suite,
-                    "BEHAT_FILTER_TAGS": "~@skipOnOcis-%s-Storage" % ("OC" if storage == "owncloud" else "OCIS"),
+                    "BEHAT_FILTER_TAGS": "~@skipOnOcis&&~@skipOnOcis-%s-Storage" % ("OC" if storage == "owncloud" else "OCIS"),
```

Comment:
why should local tests (that are only for ocis) have an `skipOnOcis` tag?

Reply:
here too, for a test running on both storage driver, if we want to skip it completely, we've to specify two tags